bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeGammaToQQGamma/VectorZPrimeGammaToQQGamma_GpT75_M10/VectorZPrimeGammaToQQGamma_GpT75_M10_run_card.dat

```diff
@@ -0,0 +1,270 @@
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  lhapdf  = pdlabel ! PDF set                                     
+$DEFAULT_PDF_SETS = lhaid
+$DEFAULT_PDF_MEMBERS = reweight_PDF     ! if pdlabel=lhapdf, this is the lhapdf number 
+
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+  False = fixed_ren_scale ! if .true. use fixed ren scale
+  False = fixed_fac_scale ! if .true. use fixed fac scale
+  91.188  = scale ! fixed ren scale
+  91.188  = dsqrt_q2fact1 ! fixed fact scale for pdf1
+  91.188  = dsqrt_q2fact2 ! fixed fact scale for pdf2
+  -1  = dynamical_scale_choice ! Choose one of the preselected dynamical choices
+  1.0 = scalefact ! scale factor for event-by-event scales
+#*********************************************************************
+# Time of flight information. (-1 means not run)
+#*********************************************************************
+  -1.0  = time_of_flight ! threshold below which info is not written
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+  1 = ickkw ! 0 no matching, 1 MLM, 2 CKKW matching
+  1 = highestmult ! for ickkw=2, highest mult group
+  1 = ktscheme ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+  1.0 = alpsfact ! scale factor for QCD emission vx
+  False = chcluster ! cluster only according to channel diag
+  False = pdfwgt ! for ickkw=1, perform pdf reweighting
+  5 = asrwgtflavor ! highest quark flavor for a_s reweight
+  True  = clusinfo ! include clustering tag in output
+  3.0 = lhe_version ! Change the way clustering information pass to shower.        
+#*********************************************************************
+#**********************************************************
+#
+#**********************************************************
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+  True  = auto_ptj_mjj ! Automatic setting of ptj and mjj
+#**********************************************************
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15.0  = bwcutoff ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*************************************************************
+  False = cut_decays ! Cut decay products 
```

Comment:
if False is used here, the cuts on the jet will not be applied on the two jets from y1 decay, just want to make sure it's you want